src/samples/audio/detect_test.c

```diff
@@ -168,6 +172,7 @@ static void default_detect_test(struct comp_dev *dev,
 		src = (valid_bits == 16U) ?
 		      audio_stream_read_frag_s16(source, sample) :
 		      audio_stream_read_frag_s32(source, sample);
+
 		if (valid_bits > 16U) {
```

Comment:
This blank line here is unrelated to the commit. If you consider this makes sense and enhances code reading you can create a separate commit later.

Reply:
Agree, will delete it. It wasn't there originally.